endpoints/openrtb2/sample-requests/valid-whole/exemplary/targeting-optional.json

```diff
@@ -0,0 +1,69 @@
+{
+    "description": "Targeting flags are all set to false, request is still valid, but not targeting data should be present in bids",
+    "config": {
+      "mockBidders": [
+        {
+          "bidderName": "appnexus",
+          "currency": "USD",
+          "price": 0.00
+        }
+      ]
+    },
+    "mockBidRequest": {
+      "id": "some-request-id",
+      "site": {
+        "page": "prebid.org"
+      },
+      "imp": [
+        {
+          "id": "some-impression-id",
+          "banner": {
+            "format": [
+              {
+                "w": 300,
+                "h": 250
+              },
+              {
+                "w": 300,
+                "h": 600
+              }
+            ]
+          },
+          "ext": {
+            "appnexus": {
+              "placementId": 12883451
+            }
+          }
+        }
+      ],
+      "tmax": 500,
+      "ext": {
+        "prebid": {
+          "targeting": {
+            "includewinners": false,
+            "includebidderkeys": false,
+            "includeformat": false
```

Comment:
The spec mentioned a specific use case of:

```
   ext.prebid.targeting: {
      includebidderkeys: false,
      includewinners: false,
      includeformat: false
   }
```

Is there an existing test covering this case? If not, please add an "exemplary" test to verify end to end functionality.